src/transaction/esdt.cpp

```diff
@@ -1,6 +1,35 @@
 #include "hex.h"
 #include "transaction/esdt.h"
 
+namespace internal
+{
+
+std::string ESDTPropertyField(std::string const &property, bool const &val)
```

Comment:
`Flag` instead of `PropertyField`? (a less generic name) - just a suggestion.

Later edit: Oh, now I see the correct name is `property` (as in docs). Can stay as it is then.

Reply:
Yes, I've named it following docs naming